### ISSUE TITLE
As a user I would like to manually search by a project

### ISSUE THREAD

**Issue by thescientist13** [ContributaryCommunity/www.contributary.community#25]
## Type of Change
- [x] New Feature Request
- [ ] Documentation
- [ ] Improvement / Suggestion
- [ ] Bug
- [ ] Other (please clarify below)

## Summary

If a user already has a name for a project, they can enter that instead.

## Details

The only downside is that this requires the user to know both the repository and repo / user name.

**Comment by thescientist13** [comment]
@penguin999 
What do you think of a task like this?  Being able to provide `repoName` / `projectName` via query string and auto populating the dropdowns that way?  (e.g. being able to deep link).  To compliment that, as the user selects from the dropdown, [maybe the URL could be updated](#47 ) for them so it could be easy for them to copy / paste and share with?

**Comment by penguin999** [comment]
@thescientist13,

> What do you think of a task like this? Being able to provide repoName / projectName via query string and auto populating the dropdowns that way?

I keep going back and forth on this one. I would keep this feature in the backlog, but it would be at low priority since we don't add much service to what the user must already know.
E.g. getting from `ContributaryCommunity/www.contributary.community` to
`https://github.com/ContributaryCommunity/www.contributary.community/issues` is not something I think a user would need to use an app for.

> To compliment that, as the user selects from the dropdown, maybe the URL could be updated for them so it could be easy for them to copy / paste and share with?

This seems to make more sense as a service: having a URL dynamically update as the user is browsing dropdown selections would communicate where things are headed, where they could directly jump to, or, as mentioned, something they could share.

**Comment by thescientist13** [comment]
> I keep going back and forth on this one. I would keep this feature in the backlog, but it would be at low priority since we don't add much service to what the user must already know.

> This seems to make more sense as a service: having a URL dynamically update as the user is browsing dropdown selections would communicate where things are headed

Yeah, I essentially see these two issues as being somewhat linked.  What I envision is a couple user flows:
- Someone can bookmark the URL generated from selecting from the dropdowns
- Someone can share that URL to someone else (like maintainer's to contributors, although I suppose maintainer's would use their repo?).  Anyway, the point is links are shareable

These both work if we make it so the app can "explode" on those query params and fast forward the user through the steps, so to speak.

With these two features, and being able to lazy load more issues, I think we would be in a good place for GitHub's hacktoberfest in October!  👻
